spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/CompositePropagationFactoryTests.java

```diff
@@ -86,6 +93,24 @@ void inject() {
 			assertThat(request).containsOnly(entry("a", "a-value"), entry("b", "b-value"));
 		}
 
+		@Test
+		void thisIsATest() {
```

Comment:
Could you please name it based on what it does?
E.g.: `shouldUseB3SingleWithParentWhenPropagationTypeIsB3`